network/src/commonMain/kotlin/com/wire/kalium/network/utils/CustomErrors.kt

```diff
@@ -0,0 +1,9 @@
+package com.wire.kalium.network.utils
+
+import com.wire.kalium.network.api.ErrorResponse
+import com.wire.kalium.network.exceptions.KaliumException
+
+object CustomErrors {
+    val MISSING_REFRESH_TOKEN =
+        NetworkResponse.Error(KaliumException.ServerError(ErrorResponse(500, "no cookie was found", "missing-refreshToken")))
```

Comment:
Maybe "missing-refreshToken" should be moved to `NetworkErrorLabel` to keep all labels in one place?